runtime/src/accounts_db.rs

```diff
@@ -2032,6 +2045,18 @@ impl AccountsDb {
         new
     }
 
+    fn init_gradual_filler_accounts(&self, slots_per_epoch: Slot) {
```

Comment:
Maybe it's worth mentioning in a comment what that the "gradual" here means? Or maybe on the field itself? Mainly somewhere that says the filler accounts will be added over the corse of an epoch's-worth of slots.